doc-content/enterprise-only/processes/using-keystore-vault-proc.adoc

```diff
@@ -0,0 +1,41 @@
+[id='using-keystore-vault-proc-{context}']
+= Using Keystore Vault
+
+You can use the Quarkiverse File Vault extension for storing credentials under the `file` method. For more information, see https://github.com/quarkiverse/quarkus-file-vault[Quarkiverse File Vault] and https://github.com/kiegroup/process-migration-service#using-keystore-vault[Using Keystore Vault] on GitHub.
+
+.Procedure
+
+. To add passwords to the Process Instance Migration (PIM) Keystore Vault, use the `keytool` command. For example:
++
+[source]
+----
+$ keytool -importpass -alias pimdb -keystore pimvault.p12 -storepass password -storetype PKCS12
+$ keytool -importpass -alias kieserver -keystore pimvault.p12 -storepass password -storetype PKCS12
+$ keytool -importpass -alias cert -keystore pimvault.p12 -storepass password -storetype PKCS12
+$ keytool -importpass -alias keystore -keystore pimvault.p12 -storepass password -storetype PKCS12
+$ keytool -importpass -alias truststore -keystore pimvault.p12 -storepass password -storetype PKCS12
+----
+. Configure the vault to use the keystore. For example:
```

Comment:
You make adding passwords the first step, configuring the vault the second step. I don't think it works like that. I think you configure the vault and THEN add passwords to it. Please check with the SME, though.

Reply:
Double-checked, your sequence is actually correct! But your formulations are confusing because the word "vault "is used in two different meanings. I would suggest not using the word "vault" at all in the first step. Instead:

In step 1, "To add passwords to a keystore file, use the keytool command. For example:"

In step 2, "configure the Process Instance Migration (PIM) Keystore Vault to use the keystore file"

Reply:
Yes, maybe it's better to clarify that the `keytool` command will add password to a new or existing keystore that the file vault extension will use.

Reply:
Thank you both. I added more detail to the section and adopted the phrasing from here.